lib/tasks/data_hygiene/export_data.rb

```diff
@@ -0,0 +1,44 @@
+module Tasks
+  module DataHygiene
+    class ExportData
+      def initialize(file, stdout)
```

Comment:
any reason we're passing stdout as a parameter here? what's the intended purpose? looking at usage below, we could do without and just use naked calls to `puts` and `print`.